docs/fsharp-core-notes.md

```diff
@@ -131,28 +129,6 @@ However, these options must be used with caution.
 
 Searching on stackoverflow reveals further guidance on this topic.
 
-### FSharp.Core in components using FSharp.Compiler.Service
```

Comment:
I guess these two sections were added because problems and questions arising with FSharp.Core were very often associated with one of these two scenarios.  It's probably worth keeping them, at least the cross-reference to the FCS docs on FSharp.Core.  

> "New versions of FSharp.Core must generally be consumable by previous generations of F# compiler tooling. There is nothing stopping older tooling from adding a reference to the new nuget package." 

This is stil factually true. In any case, a section on limitations and responsibilities for people making additions and changes to FSharp.Core seems necessary - we could in fact expand this material.   We have to document this reality somewhere, I can't think of a better place than here for now, even though it's more oriented to dotnet/fsharp developers than users.

Reply:
Isn't this implied by binary compatibility though?

Reply:
Thinking about it from the perspective of a PR review:

* Someone does something that's binary incompatible
* We notice this, request changes, and link to the top section that explains that FSharp.Core is binary compatible

What would need expanding?